src/shared/features/NewFamilyPost/NewFamilyPost.tsx

```diff
@@ -133,63 +135,76 @@ const NewFamilyPost: React.FC = () => {
+                        type="file"
+                        id="file-upload"
+                        style={{display:'none'}}
+                        onChange={(event) => onSelect(event.target.files ? event.target.files[0] : null)} />
+                </Row>
+            }
+            {selectedFile && 
+                <Row justify="center">
+                    <Button
+                        variant="outlined"
+                        color="primary"
+                        size="small"
+                        onClick={() => setSelectedFile(null)}>
+                        <ClearIcon/>Remove file
+                    </Button>
+                </Row>
+            }
+            <TextField
+                multiline
+                fullWidth
+                margin="normal"
+                rows="10"
                 variant="outlined"
-                color="primary"
-                size="small"
-                onClick={() => setSelectedFile(null)}>
-                <ClearIcon/>Remove file
+                label="Type a Message"
+                value={textValue}
+                onChange={e => updateTextValue(e.target.value)}/>
+            <Row justify="center">
+                Recipients
+            </Row>
+            {
+                receivers.map((receiver: IReceiver, index: number) => {
+                    return (
+                        <Row justify="center">
+                            <label key={receiver.id}>
+                                {receiver.name}
+                                <Checkbox name={receiver.id} checked={receiver.checked} onChange={e => handleCheckboxes(e, index)} />
+                            </label>
+                        </Row>
+                    )
+                })
+            }
+            <br/>
+            <Row justify="center">
+            <Button
+                type="submit"
+                variant="contained">
+                Send Post
             </Button>
-        }
-        <TextField
-            multiline
-            fullWidth
-            margin="normal"
-            rows="10"
-            variant="outlined"
-            label="Type a Message"
-            value={textValue}
-            onChange={e => updateTextValue(e.target.value)}/>
-        Recipients
-        <br/>
-        {
-            receivers.map((receiver: IReceiver, index: number) => {
-                return (
-                    <label key={receiver.id}>
-                        {receiver.name}
-                        <Checkbox name={receiver.id} checked={receiver.checked} onChange={e => handleCheckboxes(e, index)} />
-                    </label>
-                )
-            })
-        }
-        <br/>
-        <Button
-            type="submit"
-            variant="contained">
-            Send Post
-        </Button>
-        {error &&
-            <Alert className="error" severity="error">{error}</Alert>
-        }
-        </form>
+            </Row>
+            {error &&
```

Comment:
Can you swap this error out for the <FormError .../> component? (And any other error alerts if there are any, so they all get the same styling treatment). 

See CredentialsForm.tsx for an example, it looks like this:
```      <FormError error={error}/>```